en-replace.zh-TW.pairs

```diff
@@ -106,13 +108,15 @@ Reset
 Run (paused)
 執行（暫停）
 
+## This goes to the next command/line
 Step over
-略過一步
-#skip a step
+下一個命令
+#next command
```

Comment:
Step over & step forward are specific terms for debugging. Maybe we could find what a Chinese IDE calls them?